mods/ts/sequences/infantry.yaml

```diff
@@ -209,99 +185,97 @@ e3:
 		Length: 6
 		Facings: 8
 		ShadowStart: 456
-		Tick: 80
 	prone-shoot:
 		Start: 212
 		Length: 6
 		Facings: 8
 		ShadowStart: 504
-		Tick: 80
 	standup-0:
 		Start: 260
 		Length: 2
 		Facings: 8
 		ShadowStart: 552
-		Tick: 80
 	die5: flameguy # TODO: walking animation unused
 		Start: 42
 		Length: 104
 		ShadowStart: 190
-		Tick: 80
 	die6: electro
 		Length: *
-		Tick: 80
 	icon: e4icon
 
 weedguy:
-	stand: weed
+	Defaults: weed
```

Comment:
This makes **everything** weed, even the ones that should probably be `weedguy`, provided that is a valid image.